src/index.html

```diff
@@ -7,6 +7,318 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="logo-img"
+          class="header__logo-img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">
+              Samsung
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a class="nav__link is-active" href="#apple">
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#samsung">
              Samsung
            </a>
          </li>
```

Add blank line between multiple sibling blocks of html